lib/cstruct_cap.mli

```diff
@@ -0,0 +1,88 @@
+type 'a rd = < rd: unit; .. > as 'a
+type 'a wr = < wr: unit; .. > as 'a
+type 'a aligned = < aligned: unit; .. > as 'a
+
+type 'a t = private Cstruct_core.t
+
+type rdwr =  < rd: unit; wr: unit; >
+type ro = < rd: unit; >
+type wo = < wr: unit; >
+
+type uint8 = int
+type uint16 = int
+type uint32 = int32
+type uint64 = int64
+
+val create : int -> rdwr t
```

Comment:
does `create` zero out the memory? do we need `create_unsafe` as well (I'm ok with no `create_unsafe`)

Reply:
the functions below (`of_string` / `of_bytes`) have an `?allocator` argument -- either drop it there, or add that to `create` as well!?